src/index.js

```diff
@@ -255,13 +255,17 @@ const connectCallReceiver = (info, methods, destructionPromise) => {
  * @param {Object} [options.methods] Methods that may be called by the iframe.
  * @return {Child}
  */
-Penpal.connectToChild = ({ url, appendTo, methods = {} }) => {
+Penpal.connectToChild = ({ url, appendTo, methods = {}, sandboxOptions = [] }) => {
```

Comment:
I wonder if it won't be more future-proof if it was `iframeProperties`
Then this particular properties you want to add in current PR will be expressed as `iframeProperties.sandbox`.
If anyone wanted to apply any other properties the API won't change and it won't grow

Reply:
yet still what's in [the comment in main thread](https://github.com/Aaronius/penpal/pull/10#issuecomment-337054202) is a fair enough solution. And the basic API may stay untouched